pages.de/common/poetry.md

```diff
@@ -0,0 +1,36 @@
+# poetry
+
+> Verwalte Python-Paketen und -Abhängigkeiten.
+> Weitere Informationen: <https://python-poetry.org/docs/cli/>.
+
+- Erstellen eins neuen Poetry-Projekts im Verzeichnis mit dem angegebenem Namen:
+
+`poetry new {{projekt_name}}`
```

Comment:
```suggestion
`poetry new "{{projekt_name}}"`
```

Optional fix (the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/poetry.md) page can be updated): possible missing quotes.